schemas.md

```diff
@@ -12,34 +12,103 @@ data: one for creating a request, and another for tracking a request. The first
+- Negotiate with the requester any additional information while not required,
+  may help to narrow or expedite the request
+- Submit electronic requests directly into the agency’s case management system (if applicable)
+- Or if direct electronic submission is not possible, to email the agency with a perfected request
+
+Even though this information might already be available, it is only useful to
+the Portal if it is machine-readable. Having it in a single, standard file makes
+it easier to collect this information across agencies and components.
+
+Our research has shown that requesters often find additional requirements for
+FOIA submission confusing and burdensome. We hope that by having agencies
+include supplemental information about the requirements, it will help inform
+requesters and allay any concerns.
 
 This flow diagram illustrates how the metadata file is used for submission:
 
-![Platform reads an agency metadata file to determine how to submit a FOIA
-request to the agency](img/request-submission-platform.png)
+<img width="600"
+     alt="Portal reads an agency metadata file to determine how to submit a FOIA request to the agency"
+     src="img/request-submission-platform.png">
+
+
+### Empower agencies to keep this information up to date
+
+Having agencies responsible for maintaining this metadata file means agencies
+are empowered to change it at will, as often as their organization changes, and
+immediately when their organization changes. Agencies do not need to coordinate
+with an external entity like OIP to get this information published. We believe
+this will lead to more accurate information for requesters.
+
+In agency interviews, we offered a few options for maintaining an agency metadata file.
+
+- A **pull** approach where agencies would be responsible for creating their
+  metadata file and hosting it at a well known location on their website (e.g.
+  `agency.gov/foia.json`) from where it can be pulled.
+- A **push** approach where agencies would be responsible for creating their
+  metadata file and submitting it to a centralized repository.
+- A **GUI** approach where the metadata files would be stored in a centralized
+  repository but agencies could update their information through a web form hosted
+  at foia.gov using a secure account.
+
+The few agencies we spoke to told us that they would like the third option
+(GUI), a website where they can update their metadata file through a web UI
+rather than maintain it directly. Some of these agencies described the metadata
+file as being too technical. This has the advantage that agency FOIA offices do
+not need to coordinate with IT or other agency teams in order to update their
+metadata file. FOIA offices don’t need to be an expert in web publishing or JSON
+to update their agency’s information.
+
+
+### Schema
 
 The metadata file should include the following fields:
 
 * agency abbreviation
-* list of departments
+* list of departments/components
```

Comment:
"department" probably best to be avoided since it has specific government meaning. Maybe "list of FOIA offices/components"